branch-diff.js

```diff
@@ -111,6 +111,10 @@ function printCommits (list, format) {
   } else {
     list = list.map((commit) => commitToOutput(commit, format === 'simple', ghId))
   }
+  
+  if (reverse) {
+    list = list.reverse();
+  }
```

Comment:
That was my thinking too. If you try to apply the sha's in the correct order you are going to likely have lots of conflicts.